package.json

```diff
@@ -43,6 +43,7 @@
     ]
   },
```

Comment:
This can probably be removed, tbh.  I will investigate why this devDependency was added but it's not breaking anything.